README.md

```diff
@@ -1 +1,50 @@
 The goal of this project is to explore the network properties of T-cell repertoires.
+# code structure
+
+```
+net
+├─ singleNet.py
+├─ manyNets.py
+├─ levenshtein.py
+├─ jaccard.py
+├─ funcDictionarySingleNet.py
+├─ funcDictionaryManyNets.py
+├─ funcDictionaryLevenshtein.py
+├─ funcDictionaryJaccard.py
+├─ funcDictionary.py
+└─ plotData.py
+```
+
+This repository is still under construction and not yet finished. Please do not yet use it.
+
+The files singleNet.py, manyNets.py, levenshtein.py, jaccard.py perform network analysis and save them as txt files. 
+The use functions from dictionaries in the corresponding funcDictionarySingleNet.py, funcDictionaryManyNets.py, 
```

Comment:
I don't understand this sentence, but we can also discuss it tomorrow or in the next days. Would be good to have a clean `README.md`.